App.js

```diff
@@ -1,25 +1,77 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components/native'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
+import { Header } from './components/Header'
+import { Intro } from './components/Intro'
+import { CatList } from './components/CatList'
+import { Footer } from './components/Footer'
+
+const ScrollView = styled.ScrollView`
+  background-color: #fff5a5;
+`
+
+const Section = styled.View`
+  background-color: #fff5a5;
+  width: 100%;
+  display: flex;
   justify-content: center;
   align-items: center;
 `
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Container = styled.View`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 80%;
 `
 
 const App = () => {
+
+  const [cats, setCats] = useState([])
+  const [isLoading, setLoading] = useState(true) //remove id I don't use a loading
+
+  const url = 'https://api.thecatapi.com/v1/images/search?limit=10&page=8&order=Desc'
+
+  useEffect (() => {
+    fetch(url, {
+      headers: {
+        'x-api-key': '6935237a-0116-45be-af20-aefff1e7e239'
+      }
+    })
+    .then((response) => response.json())
+    .then((json) => setCats(json))
+    .catch((error) => console.error(error))
+    .finally(() => setLoading(false)) //remove if I don't use a loading state
+  }, [])
+
+  console.log(cats)
```

Comment:
We should remove all console.logs before final push

Reply:
removed